test_pytest_mock.py

```diff
@@ -489,3 +489,21 @@ def request(cls, method, args):
     m = mocker.patch.object(Request, 'request')
     Request.request(method='get', args={'type': 'application/json'})
     m.assert_called_once_with(method='get', args={'type': 'application/json'})
+
+
+def test_monkeypatch_native(testdir):
+    testdir.makepyfile("""
+        def test_foo(mocker):
+            stub = mocker.stub()
+            stub(1, greet='hello')
+            stub.assert_called_once_with(1, greet='hey')
+    """)
+    if hasattr(testdir, 'runpytest_subprocess'):
+        result = testdir.runpytest_subprocess('--tb=native')
+    else:
+        # pytest 2.7.X
+        result = testdir.runpytest('--tb=native')
+    assert result.ret == 1
+    assert 'Traceback (most recent call last)' in result.stdout.str()
+    assert 'During handling of the above exception' not in result.stdout.str()
+    assert 'Differing items:' not in result.stdout.str()
```

Comment:
Hmm, maybe the test should make sure there is actually no second traceback (like reported in #44)?